### PR TITLE
Rename the table formerly known as .sno-meta to gpkg_sno_state

### PR DESCRIPTION
**Pull request by olsen232**
A good time to make this breaking change since we already changed it this week, so we can just forget this intermediate change ever happened

### CODE REVIEW

File: sno/output_util.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ class ExtendedJsonLexer(
         ex_json_lexer = ExtendedJsonLexer()
         # The LexerContext stores the state of the lexer after each call to get_tokens_unprocessed
         lexer_context = pygments.lexer.LexerContext("", 0)
-        formatter = pygments.formatters.TerminalFormatter()
+        formatter = pygments.formatters.Terminal256Formatter(style="colorful")
```
Comment (craigds) [inline sno/output_util.py:64]: unrelated? what's this do? feel free to leave it in if it's useful

File: sno/working_copy.py [hunk 2]
```diff
@@ -286,11 +286,11 @@ def create(self):
 
             dbcur.execute(
                 f"""
-                CREATE TABLE {self.META_TABLE} (
+                CREATE TABLE {self.STATE_TABLE} (
                     table_name TEXT NOT NULL,
                     key TEXT NOT NULL,
                     value TEXT NULL,
-                    CONSTRAINT {self._meta_name('meta', 'pk')} PRIMARY KEY (table_name, key)
+                    CONSTRAINT {self._sno_table('meta', 'pk')} PRIMARY KEY (table_name, key)
```
Comment (craigds) [inline sno/working_copy.py:293]: constraint name doesn't really _matter_ but we should keep it consistent with the table going forward, might as well change this to `state` too

File: sno/output_util.py (hunk 1 above)
Comment (olsen232) [reply sno/output_util.py:64]: Ooops was just messing with this last night. It isn't useful

File: sno/working_copy.py (hunk 2 above)
Comment (olsen232) [reply sno/working_copy.py:293]: Done

**Pull request merged**